src/status_im/ui/screens/chats_list/views/inner_item.cljs

```diff
@@ -51,7 +51,8 @@
                (:content content)]
 
               (:command content)
-              preview
+              (update preview 1 merge {:style  st/last-message-text
```

Comment:
maybe better `assoc`?

Reply:
@flexsurfer yeah, `assoc` looks better here.

Reply:
Why not directly `assoc` ? Also is `1` a magic value? I'd rather have a dedicated small function for this.

Reply:
@trybeee agree with @jeluard it looks hacky, preview should be correctly prepared in bot , so we don't need to do something with it here, imo

Reply:
Thanks for the feedback, guys. I'm afk today, will look at it today's night hopefully.

Reply:
@flexsurfer it might be good idea to move styling to the bot short preview markup, but i'm not sure if we have access to this style setting in bot: `component.styles/text4-color` (otherwise it will copy-pasted hex `#939ba1`). Also, it sets different `height` for ios and android (not sure bot knows the platform type).

Here is the markup to apply:
https://github.com/trybeee/status-react/blob/develop/src/status_im/ui/screens/chats_list/styles.cljs#L93

Reply:
Seems like it's possible to get platform from `context.platform`. The only question is if hardcoding color code would be a good idea.

Reply:
@trybeee i think i'm fine with merging styles here, let's do something with this 1 - magic, @jeluard what do you think?

Reply:
@flexsurfer i also have new version with styles applied from bot ready.

Reply:
Just pushed commit that moves styling into bot. Please take a look @flexsurfer @jeluard